audio/psg_wave_patterns.inc.c

```diff
@@ -0,0 +1,68 @@
+
+
+/* PSG WAVE-CHANNEL PATTERNS */
+
+
+#define CREATE_PSG_WAVE(a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p)    \
```

Comment:
the names here should probably just be numbered instead of alphabetical

Reply:
I initially felt that numbered values resulted in a needlessly wide macro function definition, but honestly it's probably not as bad as I thought. There isn't really a benefit to describing the variables, since it's just a list of array elements, but I'll probably change it soon anyway.